### DISCUSSION TITLE
GID and UID (owner) different for files and folders in db folder

### DISCUSSION THREAD

**Discussion by holzhannes** [jonaswinkler/paperless-ng#1065] [Support]
I am using paperless via docker-compose and have changed the GID (USERMAP_GID) and UID (USERMAP_UID) value to cope with my setup, but I can see the folders and files inside my PostgreSQL database folder are *not* owned by the set UID and GID. Is this intentional or a mistake?

**Comment by jonaswinkler** [comment]
Yes, this is intended.

**Reply by holzhannes** [reply]
Thank you for the fast reply!

**Reply by jonaswinkler** [reply]
Just a bit more background info:

* The entire UID / GID mapping thing is performed so that the consumption folder (that's shared between host and container) can be written to from both inside and outside the container.
* This does not necessarily apply to the Postgres container. The database server inside the container is the only instance that needs to access that. Therefore, permission handling is entirely done by the postgres container, and I don't interfere with that.
* For backups, you don't need a bind mount , there are better ways to do that.
